doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -135,7 +135,7 @@ Security
     :kconfig:option:`CONFIG_TFM_PARTITION_CRYPTO` with Trusted Firmware-M (TF-M) through the
     :kconfig:option:`CONFIG_TFM_PARTITION_INTERNAL_TRUSTED_STORAGE` Kconfig option.
 
-  * Support for AES in counter mode using CRACEN for the :zephyr:board:`nrf54lm20dk`.
+  * Support for AES in counter mode and CBC mode using CRACEN for the :zephyr:board:`nrf54lm20dk`.
```

Comment:
```suggestion
  * Support for AES encryption in Counter (CTR) mode and Cipher Block Chaining (CBC) mode using CRACEN on the :zephyr:board:`nrf54lm20dk`.
```

Im guessing those acronyms. If not correct, please write their correct full name anyway